
1. Lawless D, Hodel FA, Thorball CW, Xu ZM, Borghesi A, Giannoni E, Tr√ºck J, Stocker M, Posfay-Barbe KM, Heininger U, Bernhard-Stirnemann S, Niederer-Loher A, Kahlert CR, Natalucci G, Relly C, Berger C, Riedel T, Aebi C, Agyeman P, Fellay J, Schlapbach LJ. Genome-wide association study of paediatric bacteraemia and sepsis. medRxiv 2025.06.12.25329506. doi: [https://doi.org/10.1101/2025.06.12.25329506](https://doi.org/10.1101/2025.06.12.25329506)

2. No additional columns are included. The file uses only the standard GWAS Catalogue fields: chromosome, base_pair_location, effect_allele, other_allele, beta, standard_error, effect_allele_frequency, p_value.

3. Summary statistics were generated using GCTA MLMA with a leave one chromosome out approach. Only biallelic SNPs with A, C, G, T alleles were retained. The SNP identifier column was removed to meet the GWAS Catalogue SSF format. Final formatting used the SSF-morph interface and a cleaning script. The cohort comprises Swiss paediatric bacteraemia and sepsis cases with population controls. QC included ancestry filtering, relatedness checks, variant quality filters, and standard genotype QC.

